apis/v1beta1/types.go

```diff
@@ -17,6 +17,10 @@ type ProviderConfigSpec struct {
 	// AssumeRoleChain defines the options for assuming an IAM role
 	AssumeRoleChain []AssumeRoleOptions `json:"assumeRoleChain,omitempty"`
 
+	// AssumeRoleSTSRegion defines the region used for STS requests. If not defined,
+	// the managed resource region will be used if available
+	AssumeRoleSTSRegion *string `json:"assumeRoleSTSRegion,omitempty"`
```

Comment:
nit: We may give an example of the new API under `examples/providerconfig/`, e.g., in `examples/providerconfig/user-creds-with-assume-role.yaml` or in a new example file.

Reply:
yeah definitely